scripts/roles.coffee

```diff
@@ -74,11 +84,18 @@ module.exports = (robot) ->
   robot.respond /roles shuffle/i, (msg) ->
     roles = robot.brain.get rolesKey(msg.envelope.room)
     people = robot.brain.get peopleKey(msg.envelope.room)
+    android = robot.brain.get androidKey(msg.envelope.room)
+    ios = robot.brain.get iOSKey(msg.envelope.room)
 
     if roles.length > people.length
       msg.send "Not enough people to cover all roles"
     else
       newRoles = _.object(roles, _.sample(people, roles.length))
+      newRoles = _.omit(newRoles, "🔱  Android Alchemist", "✨  iOS Illusionist")
+      newAlchemist = _.object(["🔱  Android Alchemist"], _.sample(android, 1))
+      newIllusionist = _.object(["✨  iOS Illusionist"], _.sample(ios, 1))
+
+      _.extend(newRoles, newAlchemist, newIllusionist)
```

Comment:
Hm. if other rooms are using this, it kind of forces them to use these roles in addition to what they add. Thoughts?